cloud-aws/src/main/resources/definitions/environment-minimal-policy.json

```diff
@@ -0,0 +1,82 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "cloudformation:CreateStack",
+        "cloudformation:DeleteStack",
+        "cloudformation:DescribeStacks"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeKeyPairs",
+        "ec2:CreateTags",
+        "ec2:DeleteTags",
+        "ec2:DescribeAvailabilityZones",
+        "ec2:DescribeImages",
+        "ec2:DescribeInstances",
+        "ec2:DescribeRegions",
+        "ec2:DescribeRouteTables",
+        "ec2:DescribeSecurityGroups",
+        "ec2:DescribeSubnets",
+        "ec2:DescribeVpcAttribute",
+        "ec2:DescribeVpcs",
+        "ec2:CreateVpc",
+        "ec2:CreateInternetGateway",
+        "ec2:ModifyVpcAttribute",
+        "ec2:DeleteVpc",
+        "ec2:DeleteInternetGateway",
+        "ec2:DescribeInternetGateways",
+        "ec2:AttachInternetGateway",
+        "ec2:CreateRouteTable",
+        "ec2:CreateSubnet",
+        "ec2:CreateVpcEndpoint",
+        "ec2:DescribeVpcEndpoints",
+        "ec2:DetachInternetGateway",
+        "ec2:ModifySubnetAttribute",
+        "ec2:allocateAddress",
+        "ec2:AssociateRouteTable",
+        "ec2:CreateRoute",
+        "ec2:DeleteRouteTable",
+        "ec2:DeleteSubnet",
+        "ec2:DeleteVpcEndpoints",
+        "ec2:DisassociateRouteTable",
+        "ec2:describeAddresses",
+        "ec2:releaseAddress",
+        "ec2:CreateNatGateway",
+        "ec2:DeleteRoute",
+        "ec2:DescribeNatGateways",
+        "ec2:DeleteNatGateway",
+        "ec2:ImportKeyPair"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "iam:ListInstanceProfiles"
+      ],
```

Comment:
Depending on what the purpose of this policy is, we might also need the `iam:CreateServiceLinkedRole` restricted to `arn:aws:iam::*:role/aws-service-role/*` resources. See my comment in the datahub policy.